test/application_system_test_case.rb

```diff
@@ -0,0 +1,5 @@
+require "test_helper"
+
+class ApplicationSystemTestCase < ActionDispatch::SystemTestCase
+  driven_by :selenium, using: :chrome, screen_size: [1400, 1400]
```

Comment:
We should write a sample integration test using Capybara methods!

Reply:
I used the rails command ```rails generate integration_test users``` and it generated the file ```spec/requests/users_spec.rb``` so I guess this is where rspec wants to have its integration tests. I added one example test for the sign-up process there.

Reply:
Hmm... I would not expect feature/integration tests in the `requests` folder.

Reply:
I think I agree with [this](https://www.codewithjason.com/rails-integration-tests-rspec-capybara/#whats-the-difference)

![image](https://user-images.githubusercontent.com/1652117/98384096-a0f65180-204d-11eb-9632-552fb553b879.png)

Reply:
So either `feature` (the RSpec term) or `integration` or  `endtoend` would work for me

Reply:
Request specs are a specific type of RSpec test: https://relishapp.com/rspec/rspec-rails/docs/request-specs/request-spec